ODPi-Runtime.md

```diff
@@ -154,7 +150,7 @@ The following environment variables are noted by this spec:
 | **[HADOOP_EM3]** | MAPRED_LIB_JARS_DIR | Relative Dir | Additional Apache Hadoop MapReduce jar dependencies |
 
 
--   The content of the `*_DIR` directories SHOULD be the same as the ODPi Reference Implementation and the Apache Hadoop distribution of the appropriate platform.  In a future release, this will become a MUST.
+-   The content of the `*_DIR` directories SHOULD be the same as the ODPi Reference Implementation and the binary packages made available by the upstream projects.  In a future release, this will become a MUST.
```

Comment:
-1, I disagree with Roman, I think we should use the Apache projects as the gold standard.  Otherwise we're locking ourselves to the Bigtop layout, which I don't think is the right thing to do.  

So I would make this read:  "The content of the `*_DIR` directories SHOULD be the same as the ODPi Reference Implementation and the binary packages made available by the associated Apache project.  In a future release, this will become a MUST."